extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -160,6 +160,12 @@ export class SafeframeHostApi {
     /** @private {?({width, height}|../../../src/layout-rect.LayoutRectDef)} */
     this.creativeSize_ = creativeSize;
 
+    /** @private {?({width, height}|../../../src/layout-rect.LayoutRectDef)} */
+    this.initialCreativeSize_ = {
+      height: creativeSize['height'],
+      width: creativeSize['width'],
+    };
```

Comment:
It looks like before this PR we had:
 * `creativeSize_`: initial creative size, what we `collapse()` to
 * `this.iframe_.height/width`: current creative size

And now we have:
 * `initialCreativeSize_`: initial creative size, what we `collapse()` to
 *  `creativeSize`: current creative size

It's not clear to me from the PR description and the changes here what this PR fixes.  Why do we need a new variable instead of just using `this.iframe_.height/width` for current size?  Is there a bug that this fixes, or a new capability it adds?

Reply:
this.iframe_.height/width is a string for starters, so having something that can actually be a number is nice. But I can change if needed.

Reply:
And the bug was that I erroneously thought that iframe.height was auto-updating as the height actually changes, which it does not.

Reply:
Can this go in the PR description then?

Reply:
(Including that this fixes a bug, and a little about what the bug was)

Reply:
done